libopenage/renderer/font/font.h

```diff
@@ -65,11 +65,11 @@ struct font_description {
 	 * @param language: The font's language.
 	 * @param script: The font's script.
 	 */
-	font_description(const std::string &font_file,
+	font_description(std::string font_file,
 	                 unsigned int size,
 	                 font_direction direction = font_direction::left_to_right,
-	                 const std::string &language = "en",
-	                 const std::string &script = "Latn");
+	                 std::string language = "en",
+	                 std::string script = "Latn");
```

Comment:
for default parameters, there's no spaces around the `=`. And it should say `Latin` instead of `Latn`.